CIP-0120/README.md

```diff
@@ -0,0 +1,411 @@
+iteration.
+
+### Interoperability
+
+By defining a file extension and formatting rules for the constitution we 
+ensure that tooling reading and writing the constitution will be interoperable.
+Furthermore we aim to make the role of constitution iteration comparison tools 
+easier, by minimizing formatting and style changes between iterations.
+This will reduce compatibility issues between tools, 
+promoting the accessibility of the constitution.
+
+### Usability
+
+Rich text formatting greatly enhances the readability of text, 
+especially in large complex documents.
+Without the ability to format text, it could easily become cumbersome to read, 
+negatively effecting the accessability of the Cardano constitution.
+
+## Specification
+
+The following specification SHOULD be applied to all constitution iterations.
+This standard could be augmented in the future via a separate CIP which aims to 
+replace this one.
+
+### Terminology
+
+#### Capitalisation of key terms
+
+To avoid unnecessary edits, and therefore checksum changes, 
+constitution authors MUST follow the following standard English capitalisation 
+rules unless a translation language indicates otherwise:
+
+`Constitution` vs. `constitution`
+
+- The Constitution in effect — either the "initial" one or any updated version — is unique and therefore capitalised ("Constitution") as a proper noun.
+- Draft or proposed constitutions are not unique & therefore are not capitalised ("constitution") as a common noun.
+- "Cardano Constitution" is a very specific proper noun phrase (also a title) and so each word is capitalised.
+- The phrase "the Constitution", unless used non-specifically (e.g. "the constitution that voters prefer"), would generally be assumed to be _the_ Constitution and capitalised as a proper noun.
+
+`Ada` vs. `ada` vs. `ADA`
+
+- `Ada` is the currency, while `ada` indicates units of that currency (e.g. "Ada holders can accumulate more ada to increase their influence.")
+- `ADA` is the trading symbol (e.g. "Fluctuations in ADA might influence decisions about the Treasury.")
+
+### Characters
+
+The constitution text MUST only contain UTF-8 characters.
+
+### Lines
+
+Each line in the constitution text MUST contain at maximum 80 characters,
+including spaces and punctuation.
+
+#### Sentences
+
+The constitution text MUST only contain a maximum of one sentence per group of 
+lines delimited by one blank line between these groups.
+Each sentence MUST be followed by a newline.
+
+Each sentence SHOULD start on its own line with a capitalized letter.
+
+Example:
+
+```md
+This is a short sentence on one line.
+
+This is a long sentence and I have valid reasons for it being so long,
+such as being an example of a long sentence.
+```
+
+While 80 characters is a limit, authors don't have to try and always hit 80.
+Legibility should be kept in mind.
+
+### File Type
+
+The constitution file MUST be a text file named `constitution.txt`.
```

Comment:
I think we may want versioning for the file name. I don't know if it changes anything from a technical perspective, but from a legibility perspective, over time I want to differentiate the versions of the constitution. Ex) If I download all of them into a folder to read and compare, I wouldn't want constitution.txt, constituiton.txt(1), constitution.txt(2) etc.  It would be better to have 1constitution.txt, 2constituiton.txt  etc. (in the file name itself).

Reply:
Yup!
sensible suggestion, I will put forward a change for this.

Reply:
addressed in  [1b87d84](https://github.com/cardano-foundation/CIPs/pull/796/commits/1b87d8463505e4d5943e2226a0894c20f45bcbb3)